lighthouse-core/gather/gatherers/dobetterweb/all-event-listeners.js

```diff
@@ -118,18 +118,16 @@ class EventListeners extends Gatherer {
     })).then(nestedListeners => [].concat(...nestedListeners));
   }
 
-  beforePass(options) {
```

Comment:
I have no strong opinions, are the mutation listeners bad enough that we should put them in everyone's console or just check for in LH? basically, is there a legitimate reason to still use them?